views/forms/referral_form.php

```diff
@@ -291,11 +291,11 @@
         // agencyMemberInsert for the main agency contact
         $agencyMemberResult = $db->query("SELECT agencyMemberInsert(
                                 agencyMemberID := $1::INT,
-                                agen := $2::referraltype,
-                                phn := $3::text,
-                                em := $4::text,
-                                isMain := $5::boolean,
-                                arID := $6::int
+                                agen := $2::REFERRALTYPE,
+                                phn := $3::INT,
```

Comment:
@sbohner14335 This is the part we are waiting for on the database side. This should be `phn := $3::TEXT` since their procedure had the wrong type. Same with the value on line 329.

Reply:
@blackjack26 
The fontawsome plus and minus sign are being changed in the branch https://github.com/Capping-CPCA/Web-App/tree/103-intake-referral-ui-fixes along with column widths and a few other UI fixes.